stdlib/public/core/Substring.swift

```diff
@@ -749,6 +749,39 @@ extension Substring.UTF8View: BidirectionalCollection {
   }
 }
 
+extension Substring.UTF8View {
+
+  @available(SwiftStdlib 6.2, *)
+  public var span: Span<UTF8.CodeUnit> {
```

Comment:
I think in this particular case it is particularly important for this property to have a doc comment that explains its complexity.

Unlike the other `span` properties, this one has linear time complexity for bridged strings, and that requires developers to be careful to avoid calling it in a loop. (E.g., most Substring.UTF8View operations in the stdlib (and elsewhere) must not use this property, as they have no way to avoid getting called in a loop.)